PufferyKit/Sources/PufferyUI/MessageCell.swift

```diff
@@ -52,9 +69,18 @@ extension Message.Color: View {
         }
     }
 
-    public var foregroundColor: Color? {
+    public var foregroundColor: Color {
         .white
     }
+    
+    public var secondary: Color {
+        switch self {
+        case .gray:
+            return Color(white: 0.3)
+        default:
+            return Color(white: 0.7)
+        }
+    }
```

Comment:
Great naming!

Though the gray feels a little bit "hard" to the background colors.
A transparent white would be much "softer". As you picked a good contrast with `Color(white: 0.7)`, `Color(white: 1.0, opacity: 0.7)` might fit as well.
Maybe it even fits for gray, too.